src/js/Framework/Alert.js

```diff
@@ -0,0 +1,11 @@
+export class Alert {
+  constructor () {
+    this.closeAlert()
+  }
+
+  closeAlert () {
+    setTimeout(() => {
+      $('.alert.alert-success.alert-dismissible.notification').alert('close')
```

Comment:
Hoe werkt dit precies? Want dit word on page load uitgevoerd? Ik kan mij niet direct voorstellen dat dit overal gaat werken.

Reply:
worden niet alle alerts in framework getoond on page load?

Reply:
Worden niet alle alerts in framework on page load geladen / getoond?